### PR TITLE
refactored

### PR DESCRIPTION
**Pull request by garrett-vangilder**
## Status
**READY**

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.

This PR ensures that all Session Helper instances are removed. I also removed the using statements that implemented the Session Helper into several files

## Resolves Issue Number

#122 

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.

```sh
git pull --prune
git checkout 122-remove-session-helper
dotnet run
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Look over the code, there should be no new or broken features within this branch

## Impacted Files in Application
List general components of the application that this PR will affect:

* Most Controllers

### CODE REVIEW

**Review by MattKraatz — APPROVED** [review]
LGTM

**Comment by matthamil** [comment]
LGTM

File: Helpers/SessionHelper.cs [hunk 1]
```diff
@@ -1,42 +1,45 @@
...
+//                         .AsEnumerable()
+//                         .Select(li => new SelectListItem { 
+//                             Text = $"{li.FirstName} {li.LastName}",
+//                             Value = li.UserId.ToString(),
+//                             Selected = (li.UserId == SessionHelper.ActiveUser)
+//                         });
+//             return users;
+//         } 
+//     }
+// }
```
Comment (daynewright) [inline Helpers/SessionHelper.cs]: I vote that we delete this file and folder completely from the master codebase.

**Review by daynewright — COMMENTED** [review]
Should we remove this from master since it is no longer used?

**Comment by Anulfo** [comment]
LGTM

**Pull request merged**